addons/hr_expense/migrations/9.0.2.0/post-migration.py

```diff
@@ -65,14 +65,13 @@ def hr_expense(cr):
                 (select product_id from hr_expense_line where id = %(a)s)
                 FROM hr_expense where id = %(b)s
             """ % {'a': p, 'b': expense})
-    cr.execute("""
-        UPDATE hr_expense h SET total_amount =
-        (select cast(round(unit_amount*quantity) as
-        decimal(18,2)) from hr_expense e where e.id=h.id)
-    """)
+    expenses = env['hr.expense'].search([])
+    env.add_todo(env['hr.expense']._fields['total_amount'], expenses)
+    env['hr.expense'].recompute()
 
 
 @openupgrade.migrate()
 def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
```

Comment:
how about using `use_env=True`? https://doc.therp.nl/openupgrade/API.html#openupgradelib.openupgrade.migrate
Then it will pass an environment instead of a cursor